share/functions/__fish_print_zypp_repos.fish

```diff
@@ -0,0 +1,3 @@
+function __fish_print_zypp_repos -d "Print repositories of zypper package manager"
```

Comment:
I noticed that pacman has a standalone function `__fish_print_pacman_repos`, so I thought I should put that into another file... Is it a good idea to write a function `__fish_print_repos` and do the distribution detection inside it? Just like `__fish_print_packages`?

Also thanks for such detailed explanation! Will replace `sed` with `string` soon.